cvmfs/statistics_database.cc

```diff
@@ -105,7 +110,7 @@ bool StatisticsDatabase::CreateEmptyDatabase() {
     "sz_bytes_added INTEGER,"
     "sz_bytes_removed INTEGER,"
     "sz_bytes_uploaded INTEGER,"
-    "CONSTRAINT pk_publish_statistics PRIMARY KEY (timestamp));").Execute();
+    "CONSTRAINT pk_publish_statistics PRIMARY KEY (start_time));").Execute();
```

Comment:
That could become a problem if two consecutive server commands finish in the same second, no?

Reply:
@jblomer Good point. What would be good solution: going to a finer precision or using a unique counter as primary key?

Reply:
I think I'd go for a sequence number.